app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,6 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <color name="colorBackground">#FFF9F9FF</color>
```

Comment:
just a question is there any other way to use colors like we use it in compose?

Reply:
I have used the same color background from the theme.

Reply:
I change it now to be adaptive to the theme. Thanks for your comments